tests/e2e/testnet-scripts/start-changeover.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+set -eux
+
+# The gaiad binary
+BIN=$1
+
+# JSON array of validator information
+# [{
+#     mnemonic: "crackle snap pop ... etc",
+#     allocation: "10000000000stake,10000000000footoken",
+#     stake: "5000000000stake",
+#     val_id: "alice",
+#     ip_suffix: "1",
+#     priv_validator_key: "{\"address\": \"3566F464673B2F069758DAE86FC25D04017BB147\",\"pub_key\": {\"type\": \"tendermint/PubKeyEd25519\",\"value\": \"XrLjKdc4mB2gfqplvnoySjSJq2E90RynUwaO3WhJutk=\"},\"priv_key\": {\"type\": \"tendermint/PrivKeyEd25519\",\"value\": \"czGSLs/Ocau8aJ5J5zQHMxf3d7NR0xjMECN6YGTIWqtesuMp1ziYHaB+qmW+ejJKNImrYT3RHKdTBo7daEm62Q==\"}}"
+#     node_key: "{\"priv_key\":{\"type\":\"tendermint/PrivKeyEd25519\",\"value\":\"alIHj6hXnzpLAadgb7+E2eeecwxoNdzuZrfhMX36EaD5/LgzL0ZUoVp7AK3np0K5T35JWLLv0jJKmeRIhG0GjA==\"}}"
+# }, ... ]
+VALIDATORS=$2
+
+# The chain ID
+CHAIN_ID=$3
+
+# This is the first 3 fields of the IP addresses which will be used internally by the validators of this blockchain
+# Recommended to use something starting with 7, since it is squatted by the DoD and is unroutable on the internet
+# For example: "7.7.7"
+CHAIN_IP_PREFIX=$4
+
+# A transformation to apply to the genesis file, as a jq string
+GENESIS_TRANSFORM=$5
+
+# A sed string modifying the tendermint config
+TENDERMINT_CONFIG_TRANSFORM=$6
+
+
+# CREATE VALIDATORS AND DO GENESIS CEREMONY
+# !!!!!!!!!!!!!! IMPORTANT !!!!!!!!!!!!!!!! #
+# data dir from the sovereign chain is copied to other nodes (namely bob and carol)
+# alice simply performs an upgrade
+echo "killing nodes"
+pkill -f "^"interchain-security-sd &> /dev/null || true
+
+mkdir -p /root/.sovereign/config
+
+cp -r /sover/validatoralice /.sovereign  # see: app/consumer-democracy/app.go::657
```

Comment:
Is `/.sovereign` a different directory than `/root/.sovereign`?

Reply:
True, it.
The copy step is not relevant so I removed it.